### PR TITLE
Full engine is ready

### PR DESCRIPTION
**Pull request by elinakolod**

### CODE REVIEW

File: .rake_tasks~ [hunk 1]
```diff
@@ -0,0 +1,74 @@
+app:about
```
Comment (yuriy1991) [inline .rake_tasks~]: please add to .gitignore

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# Declare your gem's dependencies in shopping_cart.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
+gemspec
+
+# Declare any dependencies that are still in development here instead of in
```
Comment (yuriy1991) [inline Gemfile]: please remove comments

File: app/views/shopping_cart/credit_cards/_form.html.haml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  .col-sm-3
+    .form-group
+      = f.label(:expiration_date, class: 'control-label input-label')
+      = f.text_field :expiration_date, id: 'firstName', class: 'form-control', placeholder: true, title: t(:invalid_exp_date), data: { regexp: INPUT_VALIDATIONS[:expiration_date] }
+  .col-sm-3
+    .form-group.relative
+      = f.label(:cvv, class: 'control-label input-label')
+      = f.text_field :cvv, id: 'cvc', class: 'form-control', placeholder: true
+      %i#info-icon.fa.fa-question-circle.general-form-help{ title: t(:card_hint) }
+
```
Comment (yuriy1991) [inline app/views/shopping_cart/credit_cards/_form.html.haml]: no new line

File: test/integration/navigation_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class NavigationTest < ActionDispatch::IntegrationTest
```
Comment (yuriy1991) [inline test/integration/navigation_test.rb]: please delete

File: app/views/shopping_cart/addresses/_address.html.haml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  = address.formatted_name
+  %span.general-edit= link_to t(:edit), wizard_path(:address) unless complete
+  %br
+  = address.address.address
+  %br
+  = address.formatted_city
+  %br
+  = address.country
+  %br
+  = address.formatted_phone
```
Comment (yuriy1991) [inline app/views/shopping_cart/addresses/_address.html.haml]: no new line

File: Gemfile (hunk 2 above)
Comment (elinakolod) [reply Gemfile]: @yuriy1991 all done

**Pull request merged**